velox/connectors/hive/tests/HiveDataSinkTest.cpp

```diff
@@ -642,28 +657,35 @@ TEST_F(HiveDataSinkTest, memoryReclaim) {
           expectedWriterReclaimEnabled,
           expectedWriterReclaimed);
     }
-  } testSettings[] = {
-      //    {dwio::common::FileFormat::DWRF, true, true, 1 << 30, true, true},
+  };
+
+  TestSetting testSettingsDwrf[] = {
+      // {dwio::common::FileFormat::DWRF, true, true, 1 << 30, true, true},
       {dwio::common::FileFormat::DWRF, true, true, 1, true, true},
       {dwio::common::FileFormat::DWRF, true, false, 1 << 30, false, false},
       {dwio::common::FileFormat::DWRF, true, false, 1, false, false},
       {dwio::common::FileFormat::DWRF, false, true, 1 << 30, true, false},
       {dwio::common::FileFormat::DWRF, false, true, 1, true, true},
       {dwio::common::FileFormat::DWRF, false, false, 1 << 30, false, false},
-      {dwio::common::FileFormat::DWRF, false, false, 1, false, false},
-  // Add Parquet with https://github.com/facebookincubator/velox/issues/5560
-#if 0
-      {dwio::common::FileFormat::PARQUET, true, true, 1 << 30, false, false},
+      {dwio::common::FileFormat::DWRF, false, false, 1, false, false}};
+  TestSetting testSettingsParquet[] = {
+      // Add Parquet with https://github.com/facebookincubator/velox/issues/5560
+      // Above issue was resolved thanks to @czentgr's PR:
+      // https://github.com/facebookincubator/velox/pull/7025 Uncommenting code
+      // below to enable PARQUET format testing
+      // {dwio::common::FileFormat::PARQUET, true, true, 1 << 30, false, false},
       {dwio::common::FileFormat::PARQUET, true, true, 1, false, false},
       {dwio::common::FileFormat::PARQUET, true, false, 1 << 30, false, false},
       {dwio::common::FileFormat::PARQUET, true, false, 1, false, false},
       {dwio::common::FileFormat::PARQUET, false, true, 1 << 30, false, false},
       {dwio::common::FileFormat::PARQUET, false, true, 1, false, false},
       {dwio::common::FileFormat::PARQUET, false, false, 1 << 30, false, false},
-      {dwio::common::FileFormat::PARQUET, false, false, 1, false, false}
-#endif
-  };
-  for (const auto& testData : testSettings) {
+      {dwio::common::FileFormat::PARQUET, false, false, 1, false, false}};
+  TestSetting(*testSettingsPtr)[7] =
```

Comment:
You should leave a space between the type and the variable.  Maybe better to write
```
TestSetting* testSettingsPtr[7]
```

The way it is currently written looks funky. And this is technically a `testSettingsPtrArray`.